{
	"about": "https://jvndb.jvn.jp/ja/contents/2022/JVNDB-2022-020112.html",
	"title": "OpenBMC における境界外読み取りに関する脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2022/JVNDB-2022-020112.html",
	"description": "OpenBMC には、境界外読み取りに関する脆弱性が存在します。",
	"identifier": "JVNDB-2022-020112",
	"references": [
		{
			"text": "https://jvn.jp/vu/JVNVU91223897/",
			"source": "JVN",
			"id": "JVNVU#91223897"
		},
		{
			"text": "https://www.cve.org/CVERecord?id=CVE-2022-35729",
			"source": "CVE",
			"id": "CVE-2022-35729"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2022-35729",
			"source": "NVD",
			"id": "CVE-2022-35729"
		},
		{
			"text": "http://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00737.html",
			"source": "関連文書",
			"id": "www.intel.com (intel-sa-00737)"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/125.html",
			"id": "CWE-125",
			"title": "境界外読み取り(CWE-125)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:openbmc-project:openbmc",
			"version": "2.2",
			"vendor": "OpenBMC",
			"product": "OpenBMC"
		}
	],
	"cvss": [
		{
			"version": "3.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"date": "2023-10-31T15:14+09:00",
	"issued": "2023-10-31T15:14+09:00",
	"modified": "2023-10-31T15:14+09:00"
}
